app/javascript/components/pages/Patientinfo.test.js

```diff
@@ -2,16 +2,28 @@ import React from "react";
 import Enzyme, { shallow } from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 import PatientInfo from "./PatientInfo";
-import fetch from "node-fetch"
+
+const mockPatient = { first_name: "Stefani", last_name: "F" };
+
+global.fetch = () =>
+  Promise.resolve({ json: () => Promise.resolve(mockPatient) });
 
 Enzyme.configure({ adapter: new Adapter() });
 
-describe("when Patientinfo  renders", () => {
-  it("display patient first name ", () => {
-    const home = shallow(<PatientInfo  />);
-    const homeHeading = home.find("h2").text();
-     expect(homeHeading).toEqual("Stefani");
+describe("when PatientInfo renders", () => {
+  it("has a button to delete the patient info", () => {
+    const info = shallow(<PatientInfo id={1} />);
+    let infoButton = info.find('a[children="Delete Patient"]')
+    expect(infoButton.length).toEqual(1);
+  });
+  it("has a button to edit the patient info", () => {
+    const info = shallow(<PatientInfo id={1} />);
+    let infoButton = info.find('a[children="Edit Patient"]')
```

Comment:
If you're not mutating the variable, `let` is less safe. In this code it really doesn't matter because there's not really a chance someone will mutated these... however, you should get into the habit now and recognize when you need to use either one.